.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,15 @@
+{
+	"name": "wadm",
+	"image": "mcr.microsoft.com/devcontainers/base:bullseye",
+	"features": {
+		"ghcr.io/devcontainers/features/rust:1": {
+			"version": "latest"
+		}
+	},
+  "containerEnv": {
+		"RUST_LOG": "INFO"
+	},
+	"postCreateCommand": "bash ./.devcontainer/install.sh",
```

Comment:
Something that I do for my dev containers is use another repo to build the image with CI, and then just reference the image here instead. I can help with that if you wanted, but if using small dev containers it shouldn't be such a significant difference. When installing a bunch with Cargo, Brew, or anything else that takes a long time it it makes a big difference. postCreateCommands will not be a part of image when you do that though, so good to note it is better to make this a local feature in those cases.

Reply:
I don't really like it being a part of the postCreate here, so I probably will just move it to a local feature. I think I did it this way because I got annoyed with the feature install being run as root and then the container couldn't find the proper binaries

Reply:
Eventually this will get a lot quicker though when we can `apt install wash` instead of building from source